cdap-app-fabric/src/main/java/io/cdap/cdap/app/preview/PreviewStatus.java

```diff
@@ -29,6 +29,8 @@
    * Status for the preview
    */
   public enum Status {
+    WAITING(false),
+    ACQUIRED(false),
```

Comment:
What is Acquired state for?

Reply:
Preview app goes to the Acquired state once the poller info is written for it. Added this state, so that if the manager restarted before INIT status is received from runner, the application is ignored while building the preview request queue.

Reply:
When do we mark a preview as ACQUIRED state? If it is during the poll call, then it means a preview can be in ACQUIRED state and yet no pod is owning it (consider the state is updated on the server, but failed to reach the client). If that happen, what will happen? I assume the preview will get timeout after sometime and marked as killed by timeout.

If that's the case, however, we can simply record it as INIT state on poll.

Reply:
Removed and using `INIT` status now.